src/main.js

```diff
@@ -103,22 +103,14 @@ function renderLocalStorageCards() {
   for (var i = 0; i < localStorage.length; i++) {
     var id = localStorage.key(i);
     var item = JSON.parse(localStorage.getItem(id));
-    ideaCardHtml += `
```

Comment:
Love how we have all this in one function to just render from the data model!  Awesome!